### PR TITLE
Feat: Added Issues templates

### PR DESCRIPTION
**Pull request by yashksaini-coder**
### Related Issue
Closes: #65

### Description
This PR fixes the issue #65. It includes issue templates that lets new contributors raise issue for particular category such as **Bug-report**, **Feature-request** and **Change-in-code** within the repository. Detailed & informative with lots of particulars to be filled that allows the maintainer to see what the contributors wants to do, their approach & how they are going to execute.

### Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

### Checklist
- [x] My code follows the code style of this project.
- [x] I have followed the contribution guidelines
- [x] I have performed a self-review of my own code.
- [x] I have ensured my changes don't generate any new warnings or errors.
- [x] I have updated the documentation (if necessary).
- [x] I have resolved all merge conflicts.

@jindalpriyanshu101 @this-is-yaash Please review my PR & add labels to it.

### CODE REVIEW

File: .github/BUG_report_issue_template.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+### Title of the Issue
+- [ ] **Have you named the issue title clearly and descriptively?**
+- [ ] **Have you searched for existing issues in the project to avoid duplicates?**  
+<!-- Example: "fix: [Component] Button not responding on click" -->
+<!-- Use format: "fix: [affected section] concise description" -->
```
Comment (jindalpriyanshu101) [inline .github/BUG_report_issue_template.md:5]: Isn't it should be `BUG: `?

File: .github/Changes_in_code_template.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (jindalpriyanshu101) [inline .github/Changes_in_code_template.md:70]: Delete these empty lines

**Review by jindalpriyanshu101 — CHANGES_REQUESTED** [review]
I have added a few changes, please implement them.

File: .github/Changes_in_code_template.md (hunk 2 above)
Comment (yashksaini-coder) [reply .github/Changes_in_code_template.md:70]: sorry about this

File: .github/BUG_report_issue_template.md (hunk 1 above)
Comment (yashksaini-coder) [reply .github/BUG_report_issue_template.md:5]: okh

**Comment by yashksaini-coder** [comment]
@this-is-yaash Okh done all changes you can review it now.

**Review by jindalpriyanshu101 — APPROVED** [review]
Everythin Looks good to me! Great commit.

**Comment by yashksaini-coder** [comment]
> Everythin Looks good to me! Great commit.

thx @jindalpriyanshu101

**Pull request merged**